edit-post/components/header/writing-menu/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress Dependencies
+ */
+import { MenuGroup } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { ifViewportMatches } from '@wordpress/viewport';
+
+/**
+ * Internal dependencies
+ */
+import FixedToolbarToggle from '../fixed-toolbar-toggle';
+
+function WritingMenu( { onClose } ) {
+	return (
+		<MenuGroup
+			label={ __( 'Writing' ) }
+			filterName="editPost.MoreMenu.writing"
```

Comment:
You can remove this filter, it shouldn't be used in the long run. We should rather offer some fill if we want to make it possible to add item to this group.

Reply:
I was going to ping you about it :)